docs/poetry.lock

```diff
@@ -1,25 +1,27 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.1.2 and should not be changed by hand.
```

Comment:
We still use poetry 1.8.3, trying to update (see https://github.com/scylladb/scylla-operator/blob/f8edb3bee935e3c778a749e9d7abcb8acde44620/docs/README-dev.md#update-dependencies) with these changes results in this warning:
```
The lock file might not be compatible with the current version of Poetry.
Upgrade Poetry to ensure the lock file is read properly or, alternatively, regenerate the lock file with the `poetry lock` command.
```

Not really sure if this is an issue but I'd stick to using the same version.

Reply:
Done!